Analytics/gunicornserver.py

```diff
@@ -0,0 +1,51 @@
+from flask_script import Command, Option
+
+class GunicornServer(Command):
+
+    description = 'Run the app within Gunicorn'
+
+    def __init__(self, host='api', port=5000, workers=4):
```

Comment:
@Aibaki The parameter `host='api'` is only applicable to the docker specific way of working, but not the ubuntu deployment.

Please specify this in the deployment doc for both deployment methods and identify as a conditional value, perhaps using `host='<host-value>'`.